client/style.css

```diff
@@ -647,4 +655,116 @@ body {
   padding-left: 0;
   width: 50%;
   float: left;
+}
+
+.modal-open  {
+  overflow: hidden;
+  padding-right: 15px;
+  background-color: white;
+  height: 100%;
+}
+.fade {
+  opacity: 0;
+  transition: opacity 0.15s linear;
+}
+.modal-open .modal {
+  overflow-x: hidden;
+  overflow-y: auto;
+}
+.modal {
+  display: none;
+  overflow: hidden;
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: -1;
```

Comment:
o.0 does this make you modal under the page?

Reply:
yeah i think this builds it and then hides it below the page